docs/naps/6-contributable-menus.md

```diff
@@ -238,6 +238,9 @@ Layers
 │  ├─ Tracks
 │  ├─ Classification
 Acquisition
+Plugins
```

Comment:
I wouldn't add this here as the `Plugins` menu itself is not open for contribution - only the individual plugin's menu within there would be.

Reply:
Good point, what about expanding to:
`Plugins/<my_plugin>    # only individual plugins submenu contributable

?

Reply:
yep @lucyleeow that works!